sections/Discussion.tex

```diff
@@ -8,10 +8,38 @@ \section{Discussion}\label{sec:discussion}
 The article is about agriculture and how farmers are letting their cows out onto grass fields in September. 
 It also mentions a few different farms in the Northern part of Jutland and describes these in various ways.
 
-The way we can compare these models, is by taking the top 200 words for each topic-word distribution within each model and marking them in the article below. \vejleder[inline]{why 200?}
+\begin{table}
+	\begin{tabular}{c|p{0.8\columnwidth}}
+		Topic & Top 10 words for \gls{lda} \\
+		\toprule
+		0 & 'sæby', 'a', 'direktør', 'frederikshavn', 'virksomheden', 'hans', 'medarbejdere', 'pedersen', 'firmaet', 'procent' \\
+		1 & 'omradet', 'boliger', 'natur', 'naturen', 'ligger', 'du', 'vand', 'dyr', 'a', 'skov' \\
+		2 & 'nielsen', 'arets', 'prisen', 'dansk', 'mors', 'jensen', 'løgstør', 'aars', 'vm', 'thy' \\
+	\end{tabular}
+	\begin{tabular}{c|p{0.8\columnwidth}}
+		Topic & Top 10 words for Author-Topic \\
+		\toprule
+		0 & 'du', 'procent', 'unge', 'børn', 'arige', 'hans', 'dansk', 'mig', 'thisted', 'mener'\\
+		1 & 'sine', 'skriver', 'mig', 'børn', 'seneste', 'land', 'dansk', 'kommuner', 'andersen', 'formand' \\
+		2 & 'set', 'glas', 'odense', 'vesthimmerland', 'leth', 'markedet', 'trump', 'ni', 'regionerne', 'prins'\\
+	\end{tabular}
+	\begin{tabular}{c|p{0.8\columnwidth}}
+		Topic & Top 10 words for Category-Topic \\
+		\toprule
+		0 & 'du', 'hans', 'børn', 'mig', 'thisted', 'procent', 'bedre', 'a', 'kr', 'maske' \\
+		1 & 'klar', 'haft', 'fem', 'hjørring', 'ham', 'nyt', 'formand', 'min', 'aften', 'sagen'\\
+		2 & 'du', 'min', 'gode', 'gamle', 'ad', 'henrik', 'eu', 'finde', 'sat', 'hobro'\\
+	\end{tabular}
+	\caption{Top 10 words for each model used in \autoref{tab:disc_color}}
+	\label{tab:top_words_three_models}
+\end{table}
+
+In \autoref{tab:top_words_three_models}, we can see that there is not many top words in the Author-topic and Category-Topic model that appear in the article below.
+In order to compare these models, we take the top 200 words for each topic-word distribution within each model and marking them in the article below.
+We have taken 200 words, since we want to see how intertwined the models, but it is hard to find specific words since the topic coherence is low for Author and Category-Topic model.
```

Comment:
```suggestion
We take 200 words since we want to see how intertwined the models are and because it is hard to find specific words for the Author and Category-Topic model since the topic coherence is low.
```